scripts/mbedtls_dev/bignum_mod_raw.py

```diff
@@ -49,6 +49,25 @@ def result(self) -> List[str]:
         result = (self.int_a - self.int_b) % self.int_n
         return [self.format_result(result)]
 
+class BignumModRawMul(bignum_common.ModOperationCommon,
+                      BignumModRawTarget):
+    """Test cases for bignum mpi_mod_raw_mul()."""
+    symbol = "*"
+    test_function = "mpi_mod_raw_mul"
+    test_name = "mbedtls_mpi_mod_raw_mul"
+    input_style = "arch_split"
+    arity = 2
+
+    def arguments(self) -> List[str]:
+        return [bignum_common.quote_str(n) for n in [self.arg_a,
+                                                     self.arg_b,
+                                                     self.arg_n]
+               ] + self.result()
+
+    def result(self) -> List[str]:
+        result = (self.int_a * self.int_b) % self.int_n
+        return [self.format_result(result)]
```

Comment:
Since `mbedtls_mpi_mod_raw_mul()` only takes input in Montgomery form, would it be simpler to have the Python calculate that and not do it in the `.function`?

Reply:
That's a good catch. The test script's conversion functions have been merged. You can use [self.to_montgomery()](https://github.com/Mbed-TLS/mbedtls/blob/development/scripts/mbedtls_dev/bignum_common.py#L256-L257) to convert the data and the expected return greatly simplifying the .function